src/upasana/testCases/TestCase10.java

```diff
@@ -0,0 +1,41 @@
+/*
+Name -> "Maulik", print lastName and UserName
+ */
+
+package upasana.testCases;
+
+import java.util.HashMap;
+
+import java.util.concurrent.TimeUnit;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+import base.PredefinedActions;
+
+public class TestCase10 {
+
+	static HashMap<String, String> getOutput(WebDriver driver) {
```

Comment:
If you create driver at instance level so we not require to pass reference of driver in every method